anx/provider/loadbalancer/reconciliation/reconcile_binds.go

```diff
@@ -12,27 +12,27 @@ import (
 
 const bindResourceTypeIdentifier = "bd24def982aa478fb3352cb5f49aab47"
 
-func (r *reconciliation) storePublicAddress(addr string) {
-	if idx := sort.SearchStrings(r.publicAddresses, addr); idx >= len(r.publicAddresses) || r.publicAddresses[idx] != addr {
-		r.publicAddresses = append(r.publicAddresses, addr)
-		sort.Strings(r.publicAddresses)
+func (r *stateRetrieverImpl) storePublicAddress(lbID, addr string) {
```

Comment:
maybe this should move to state_retriever.go now? And do we need the public IPs to be per LoadBalancer?

Reply:
It appears that `publicAddresses` isn't used at all. I think it became obsolete with #46.

Reply:
It didn't exist before #46 though? But yeah, seems to be unused - wonder what I planned to do with it ... :D

Reply:
Probably wanted to use it for LoadBalancer status, to report the public address back into the cluster - but then realized I have to check all the ports, too, so now iterating over the `Binds` anyway

Reply:
I've removed code related to `publicAddresses`.